src/wizard.scss

```diff
@@ -0,0 +1,300 @@
+@import "./mixins";
+
+// VARIABLES
+$block: #{$fd-namespace}-wizard;
+
+$fd-wizard-step-indicator-spacing: 0.1875rem !default;
+$fd-wizard-step-indicator-size: 2rem !default;
+$fd-wizard-step-no-label-max-size: 4rem !default;
+$fd-wizard-step-indicator-stacked-offset: -1 * ($fd-wizard-step-indicator-size - $fd-wizard-step-indicator-spacing + 0.25rem) !default;
+$fd-wizard-stacked-upcoming-step-offset: 2.375rem !default;
+
+@mixin fd-wizard-responsive-paddings() {
+  &--s {
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  &--m,
+  &--l {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  &--xl {
+    padding-left: 3rem;
+    padding-right: 3rem;
+  }
+}
+
+// BLOCK
+.#{$block} {
+  @include fd-reset();
+
+  position: relative;
+
+  &__navigation {
+    @include fd-reset();
+  }
+
+  &__progress-bar {
+    @include fd-reset();
+    @include fd-wizard-responsive-paddings();
+
+    width: 100%;
+    height: 4rem;
+    display: table;
+    padding-top: 1rem;
+    position: relative;
+    table-layout: fixed;
+    list-style-type: none;
+    background: var(--sapObjectHeader_Background);
+    box-shadow: var(--sapContent_HeaderShadow); // to be discussed
+  }
+
+  &__step-wrapper {
+    @include fd-reset();
+    @include fd-flex-vertical-center();
+
+    height: 2rem;
+  }
+
+  &__step {
+    @include fd-reset();
+
+    width: 100%;
```

Comment:
![image](https://user-images.githubusercontent.com/26483208/93208472-e9377880-f75c-11ea-99b7-025a89c8e89a.png)
Last one without any line has same width as rest of them. I'm not sure if it's correct behaviour

Reply:
<img width="474" alt="Screen Shot 2020-09-15 at 9 31 22 AM" src="https://user-images.githubusercontent.com/39598672/93216873-3f44f580-f736-11ea-9ff3-91a32c47aadd.png">

That's because it's following the specs and has a min-width. In this specific case, since there are multiple steps, their width is equal and equal to the min-width.